blocks/v2-media-with-text/v2-media-with-text.js

```diff
@@ -0,0 +1,55 @@
+import {
+  addVideoToSection,
+  createElement,
+  createNewSection,
+  unwrapDivs,
+  variantsClassesToBEM,
+} from '../../scripts/common.js';
+
+const blockName = 'v2-media-with-text';
+const variantClasses = ['text-centered', 'full-width', 'media-left', 'media-left', 'media-right', 'media-vertical', 'media-gallery'];
+export default async function decorate(block) {
+  variantsClassesToBEM(block.classList, variantClasses, blockName);
+
+  const isFullWidthBlock = block.className.includes('full-width');
+  if (isFullWidthBlock) block.parentElement.classList.add('full-width');
+
+  const cells = block.querySelectorAll(':scope > div > div');
+  let contentSection; let mediaSection; let subTextSection; let
+    containerSection;
+
+  cells.forEach((cell, index) => {
+    // First cell for content, second for media and last for the subtext
+    const isLastCell = index % 2 === 0 && index === cells.length - 1;
```

Comment:
From the comment above I assume that the cell array can contain max 3 elements. So why are we checking so many conditions `isLastCell`, 'isCellNumberEven`, and `isTotalCellEven`?
Why don't we check if the index is 0, 1, or 2? LIke:
```
const isLast = cells.length === index;

// subText section
if (index === 2) {
    subTextSection = ...;
    
    
} else if (index === 1) {
    // media section
} else {
    //text
}

if (isLast && index === 2)

```

In this case, I would prefer indexes because we are limited to fixed width. And it is clear what we are expecting in which cell. Checking if the cell is even in this case makes me expect a long list where every even item is treated differently from odd items.

Reply:
@TomaszDziezykNetcentric thanks for pointing this out. There can be more elements than 3 that's why i avoided magic numbers.